components/javascript/src/main/java/org/datacleaner/beans/script/JavaScriptTransformer.java

```diff
@@ -52,9 +52,7 @@
 	private static final Logger logger = LoggerFactory
 			.getLogger(JavaScriptTransformer.class);
 
-	public static enum ReturnType {
-		STRING, NUMBER, BOOLEAN;
-	}
+	public enum ReturnType {STRING, NUMBER, BOOLEAN}
```

Comment:
Might just be my old C ways, but I'd never use the "class" format for Enums that are just simple enumerations without behavior.